packages/migrate/src/MigrateEngine.ts

```diff
@@ -135,11 +142,29 @@ export class MigrateEngine {
     return this.runCommand(this.getRPCPayload('evaluateDataLoss', args))
   }
 
+  public getDatabaseDescription(schema: string): Promise<string> {
+    return this.runCommand(this.getRPCPayload('getDatabaseDescription', { schema }))
+  }
+
   /**
    * Get the database version for error reporting.
    */
-  public getDatabaseVersion(): Promise<string> {
-    return this.runCommand(this.getRPCPayload('getDatabaseVersion', undefined))
+  public getDatabaseVersion({ schema }: EngineArgs.GetDatabaseVersionParams): Promise<string> {
+    return this.runCommand(this.getRPCPayload('getDatabaseVersion', { schema }))
```

Comment:
Why does this now also take/need the schema?

Reply:
The `getDatabaseVersion` RPC method had always required a schema. We didn't notice that the `getDatabaseVersion` recently added to MigrateEngine was missing an input argument because that method wasn't used in tests